host/traceh.c

```diff
@@ -310,17 +314,18 @@ oe_result_t oe_log(oe_log_level_t level, const char* fmt, ...)
 void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
 {
     // get timestamp for log
+    struct tm t;
 #if defined(__linux__)
     struct timeval time_now;
     gettimeofday(&time_now, NULL);
-    struct tm* t = gmtime(&time_now.tv_sec);
+    gmtime_r(&time_now.tv_sec, &t);
 #else
     time_t lt = time(NULL);
-    struct tm* t = gmtime(&lt);
+    gmtime_s(&t, &lt);
```

Comment:
Should not need two #if cases here.  Include internal/datetime.h which games gmtime_r ubiquitous.

Reply:
But in the `if` part, there is no `gettimeofday()` for Windows. Or should I just stick to the `else` part?